File: 2st/Yeonwoo/숫자 찾기.py
def solution(num, k):
    answer = 0
    str_num = str(num)
    str_k = str(k)
    if str_k in str_num:
        for i in str_num:
            if str_k == i:
                answer = str_num.find(i)+1
    else:
        answer = -1
    return answer

File: 4st/Seohyun/programmer/숫자 찾기.java
class Solution {
    public int solution(int num, int k) {
        String str = String.valueOf(num);
        String find = String.valueOf(k);
        
        for(int i=0; i<str.length(); i++) {
            if(str.charAt(i) == find.charAt(0)) {
                return i+1;
            }
        }
        return -1;
    }
}